arelle/plugin/validate/NL/rules/fg_nl.py

```diff
@@ -92,3 +97,38 @@ def rule_fg_nl_04(
             beforeName=qname(beforeName),
             afterName=qname(afterName),
         )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NT16,
+        DISCLOSURE_SYSTEM_NT17,
+        DISCLOSURE_SYSTEM_NT18,
+    ],
+)
+def rule_fg_nl_11(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation] | None:
+    """
+    FG-NL-11: XBRL instance file Should only contain standard units based on the XBRL units Registry.
+    """
+    modelXbrl = val.modelXbrl
+    customUnits = []
+    for unitId, unit in modelXbrl.units.items():
+        for measures in unit.measures:
+            for measure in measures:
+                if measure.namespaceURI not in VALID_UTR_NAMESPACES:
+                    customUnits.append(unit)
+    if customUnits:
+        for unit in customUnits:
+            yield Validation.warning(
+                codes='NL.FG-NL-11',
+                msg=_('"Default units SHOULD be used in measure elements, a.k.a. should not be using a custom unit. '
```

Comment:
The leading `"` looks wrong.

"Default unit" is an unfamiliar term, and "also known as" seems like odd phrasing.  Perhaps it should just say what the manual says: "Standard units SHOULD be used"